web/views_avatar.py

```
@@ -138,6 +145,90 @@ def customize_avatar(request):
+                profile.save(update_fields=["custom_avatar"])
+
+            # Clear old avatar reference (file deleted after commit)
+            if old_avatar:
+                profile.avatar = None
+                profile.save(update_fields=["avatar"])
+
+            # Save new avatar
+            profile.avatar = avatar_file
+            profile.save(update_fields=["avatar"])
+
+            def delete_old_files():
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `delete_old_files`

Add return type annotation: `None`

(ANN202)